wallet/src/Pos/Wallet/Web/Account.hs

```diff
@@ -95,16 +96,14 @@ getSKByAddressPure secrets scp passphrase addrMeta = do
 genSaveRootKey
     :: (AccountMode ctx m, MonadKeys m)
     => PassPhrase
-    -> BackupPhrase
+    -> Mnemonic
     -> m EncryptedSecretKey
-genSaveRootKey passphrase ph = do
-    sk <- either keyFromPhraseFailed (pure . fst) $
-        safeKeysFromPhrase passphrase ph
-    addSecretKey sk
-    return sk
-  where
-    keyFromPhraseFailed msg =
-        throwM . RequestError $ "Key creation from phrase failed: " <> msg
```

Comment:
This will potentially break Deadelus if it depends on parsing this error to handle UI.

First iteration of Daedalus parsed string error msgs to figure out what went wrong, I am not sure was it changed recently. 

EDIT: quick search in deadalus repo didn't reveal anything so I guess this error is handled internally by our backend (or msg is changed - or its not handled). Anyway, looks ok to me

Reply:
We should not expect that users are handling stringly-typed errors.